torch/jit/operator_upgraders.py

```diff
@@ -0,0 +1,102 @@
+"""
+This is the centralized file for all PyTorch operator upgraders.
+Each function definition here needs to following requirements:
+1. The body of the function must be Torchscriptable
+2. The naming convention of the upgraders should be:
+    <op_name>_<op_overload>_upgrader_<old_version>_<new_version>
+3. The name of the upgrader must be present in operator_versions.yaml
+"""
+import torch
+import yaml
+from typing import Union, List, Optional
+
+@torch.jit.script
+def div_Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
+    if (self.is_floating_point() or other.is_floating_point()):
+        return self.true_divide(other)
+    return self.divide(other, rounding_mode='trunc')
+
+@torch.jit.script
+def div_Scalar_0_3(self: torch.Tensor, other: Union[int, float, complex]) -> torch.Tensor:
+    if (self.is_floating_point() or isinstance(other, float)):
+        return self.true_divide(other)
+    return self.divide(other, rounding_mode='trunc')
+
+# # TODO: not present in the schema
```

Comment:
I am assuming its safe to not have upgraders here. The current version of this function does exact same thing as this one here. (@gmagogsfm)

Reply:
"The current version of this function does exact same thing as this one here."

What are "this function" and "this one"?

Reply:
This function (aka operator) -> the upgrader for `"def div_0_3(self: number, other: number) -> number"`

this one -> (https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/register_prim_ops.cpp#L806) (the current version of this operator)

Reply:
Yeah, they do seem identical. `/` is compiled to `aten::div` anyway, so this is a no op. Wonder why this was added in the first place.

Reply:
Resolved offline